generator/carbonMonoxideSensor/main.go

```diff
@@ -0,0 +1,94 @@
+package main
```

Comment:
Please use Artem's work in `master` to run your generator simultaneously with others